formatter.go

```diff
@@ -34,6 +34,9 @@ type Formatter struct {
 	// ShowFullLevel - show a full level [WARNING] instead of [WARN]
 	ShowFullLevel bool
 
+	// NoUpcaseLevel - no upper case for level value
+	NoUpcaseLevel bool
```

Comment:
Hey @dfateyev! Thanks for the PR, I like it! Let's make it `NoUppercaseLevel`?

Reply:
> Let's make it `NoUppercaseLevel`?

Well, the only concern is it would be bulky being compared to other options (16 chars against the longest 14 chars `NoFieldsColors` parameter). But if you think it's better — I can adjust the name.

Reply:
Yes, it's longer, but it feels grammatically correct :+1:

Reply:
I have merged all changes into one commit, and renamed PR according the new option's name. Please review the current changes.

Reply:
```
	NoColors bool

	// NoFieldsColors - apply colors only to the level, default is level + fields
	NoFieldsColors bool

	// NoFieldsSpace - no space between fields
	NoFieldsSpace bool

	// ShowFullLevel - show a full level [WARNING] instead of [WARN]
	ShowFullLevel bool

	// NoUppercaseLevel - no upper case for level value
	NoUppercaseLevel bool

	// TrimMessages - trim whitespaces on messages
	TrimMessages bool
```
Btw, "NoUppercaseLevel" is placed after "ShowFullLevel" in Readme. In case if it matters — I can move it after "NoFieldsSpace" (to me, there is no much difference though).